lib/shared/log/tasks.js

```diff
@@ -13,7 +13,23 @@ function isObject(val) {
   return typeof val === 'object' && !Array.isArray(val);
 }
 
+function wrapGetTask(getTask) {
```

Comment:
this probably shouldn't be in the shared module and should instead go in the `getTask` methods of alpha.2 and 4.0